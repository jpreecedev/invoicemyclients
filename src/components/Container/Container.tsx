import React from 'react';
import clsx from 'clsx';

import styles from './styles.module.css';

interface ContainerProps {
	additionalClasses?: string | string[];
}

const Container: React.FC<ContainerProps> = ({ children, additionalClasses = '' }) => {
	return (
		<section className={clsx(styles.container, 'shadow-sm mb-5 rounded-lg', additionalClasses)}>
			{children}
		</section>
	);
};

export { Container };
