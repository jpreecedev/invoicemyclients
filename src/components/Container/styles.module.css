.container {
  padding: 0.5rem;
  background-color: #fdfdfd;
  border: 1px solid #f8f9fa;
}

@media screen and (min-width: 768px) {
  .container {
    padding: 1rem;
  }
}
