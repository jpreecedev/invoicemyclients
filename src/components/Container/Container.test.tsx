import React from 'react'
import { render } from '@testing-library/react'
import { Container } from './Container'

describe('<Container /> tests', () => {
  it('should match snapshot', () => {
    expect(render(<Container />).container).toMatchSnapshot()
  })
})
