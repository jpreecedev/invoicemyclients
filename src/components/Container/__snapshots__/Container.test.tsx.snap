// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> tests should match snapshot 1`] = `
<div>
  <section
    class="container shadow-sm mb-5 rounded-lg"
  />
</div>
`;
